feat(vllm): Adding inference example for vLLM (#417)
coreweave/kubernetes-cloud: online-inference/vllm/README.md

This folder contains instructions to run the vLLM inference server.

Some of the features include:

> 1. Serialize HuggingFace models supported here to vLLM based format: https://github.com/vllm-project/vllm?tab=readme-ov-file#about

Reviewer: ```suggestion
1. Serialize a [vLLM-supported model](https://github.com/vllm-project/vllm?tab=readme-ov-file#about) from the HuggingFace Model Hub.
```
